shippers/es.py

```diff
@@ -32,6 +33,7 @@ def __init__(
         dataset: str = "",
         namespace: str = "",
         tags: list[str] = [],
+        _es_index: str = "",
```

Comment:
`_es_index` is not defined in the output, or do I miss anything?
so no need to add it in `__init__` as for https://github.com/elastic/elastic-serverless-forwarder/blob/main/shippers/factory.py#L35-L44

Reply:
removed and refactored :)